homeassistant/components/lg_thinq/sensor.py

```diff
@@ -0,0 +1,590 @@
+        key=ThinQProperty.HOOD_OPERATION_MODE,
+        translation_key=ThinQProperty.OPERATION_MODE,
+    ),
+    ThinQProperty.WATER_HEATER_OPERATION_MODE: SensorEntityDescription(
+        key=ThinQProperty.WATER_HEATER_OPERATION_MODE,
+        translation_key=ThinQProperty.OPERATION_MODE,
+    ),
+}
+POWER_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.POWER_LEVEL: SensorEntityDescription(
+        key=ThinQProperty.POWER_LEVEL,
+        translation_key=ThinQProperty.POWER_LEVEL,
+    )
+}
+PREFERENCE_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.RINSE_LEVEL: SensorEntityDescription(
+        key=ThinQProperty.RINSE_LEVEL,
+        translation_key=ThinQProperty.RINSE_LEVEL,
+    ),
+    ThinQProperty.SOFTENING_LEVEL: SensorEntityDescription(
+        key=ThinQProperty.SOFTENING_LEVEL,
+        translation_key=ThinQProperty.SOFTENING_LEVEL,
+    ),
+    ThinQProperty.MACHINE_CLEAN_REMINDER: SensorEntityDescription(
+        key=ThinQProperty.MACHINE_CLEAN_REMINDER,
+        translation_key=ThinQProperty.MACHINE_CLEAN_REMINDER,
+    ),
+    ThinQProperty.SIGNAL_LEVEL: SensorEntityDescription(
+        key=ThinQProperty.SIGNAL_LEVEL,
+        translation_key=ThinQProperty.SIGNAL_LEVEL,
+    ),
+    ThinQProperty.CLEAN_LIGHT_REMINDER: SensorEntityDescription(
+        key=ThinQProperty.CLEAN_LIGHT_REMINDER,
+        translation_key=ThinQProperty.CLEAN_LIGHT_REMINDER,
+    ),
+}
+RECIPE_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.RECIPE_NAME: SensorEntityDescription(
+        key=ThinQProperty.RECIPE_NAME,
+        translation_key=ThinQProperty.RECIPE_NAME,
+    ),
+    ThinQProperty.WORT_INFO: SensorEntityDescription(
+        key=ThinQProperty.WORT_INFO,
+        translation_key=ThinQProperty.WORT_INFO,
+    ),
+    ThinQProperty.YEAST_INFO: SensorEntityDescription(
+        key=ThinQProperty.YEAST_INFO,
+        translation_key=ThinQProperty.YEAST_INFO,
+    ),
+    ThinQProperty.HOP_OIL_INFO: SensorEntityDescription(
+        key=ThinQProperty.HOP_OIL_INFO,
+        translation_key=ThinQProperty.HOP_OIL_INFO,
+    ),
+    ThinQProperty.FLAVOR_INFO: SensorEntityDescription(
+        key=ThinQProperty.FLAVOR_INFO,
+        translation_key=ThinQProperty.FLAVOR_INFO,
+    ),
+    ThinQProperty.BEER_REMAIN: SensorEntityDescription(
+        key=ThinQProperty.BEER_REMAIN,
+        native_unit_of_measurement=PERCENTAGE,
+        translation_key=ThinQProperty.BEER_REMAIN,
+    ),
+}
+REFRIGERATION_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.FRESH_AIR_FILTER: SensorEntityDescription(
+        key=ThinQProperty.FRESH_AIR_FILTER,
+        translation_key=ThinQProperty.FRESH_AIR_FILTER,
+    ),
+    ThinQProperty.ONE_TOUCH_FILTER: SensorEntityDescription(
+        key=ThinQProperty.ONE_TOUCH_FILTER,
+        translation_key=ThinQProperty.ONE_TOUCH_FILTER,
+    ),
```

Comment:
What does this do?

Reply:
In case of Kimchi refrigerator>

Air filter>
Off / Auto / Power
On touch filter>
On/Off

Btw, I love Kimchi.